stdlib/src/zlib.rs

```diff
@@ -142,18 +145,18 @@ mod zlib {
     }
 
     #[derive(Clone)]
-    struct Chunker<'a> {
+    pub(crate) struct Chunker<'a> {
         data1: &'a [u8],
         data2: &'a [u8],
     }
     impl<'a> Chunker<'a> {
-        fn new(data: &'a [u8]) -> Self {
+        pub(crate) fn new(data: &'a [u8]) -> Self {
```

Comment:
```suggestion
        pub fn new(data: &'a [u8]) -> Self {
```
When the struct is `pub(crate)`, only `pub` here is automatically `pub(crate)`

Reply:
True, I think I'll change that in the follow-up.